.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
```

Comment:
i think .env*.local includes .env but maybe some one can confirm

Reply:
No it doesn't include `.env`.
It's better to include it but in any case, working with `.env.local` should be sufficient during development. `.env` keys and values have to be manually uploaded in the server provider,  e.g. _Vercel_.

Reply:
Just came up with that while trying around, deleted it again for the moment as for now it's not yet relevant. But thanks for the insights, probably useful for later.